lib/components/map/enhanced-stop-marker.js

```diff
@@ -23,43 +23,42 @@ const { ViewStopButton } = StopsOverlayStyled
 const getComplementaryColor = (color) =>
   color.isLight() ? color.darken(30) : color.lighten(40)
 
-const caretPixels = 24
+const caretPixels = 18
 const iconPixels = 32
-const iconPadding = 8
-const borderPixels = (props) => (props?.thick ? 3 : 1)
-const caretVisibleHeight = caretPixels / 1.4142 // Math.sqrt(2)
+const iconPadding = 5
+const borderPixels = (props) => (props?.active ? 3 : 1)
 const caretMarginPixels = (props) =>
   (iconPixels - caretPixels - borderPixels(props)) / 2
-const bubblePixels = (props) =>
-  iconPixels + iconPadding + 2 * borderPixels(props)
+
+const defaultColor = '#a6a6a6'
```

Comment:
Perhaps these constants should be all caps?

Reply:
Good idea! Refactored